src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2084,7 +2081,7 @@ namespace winrt::TerminalApp::implementation
                         {
                             autoPeer.RaiseNotificationEvent(Automation::Peers::AutomationNotificationKind::ActionCompleted,
                                                             Automation::Peers::AutomationNotificationProcessing::ImportantMostRecent,
-                                                            fmt::format(std::wstring_view{ RS_(L"TerminalPage_PaneMovedAnnouncement_ExistingWindow") }, tabTitle, windowId),
```

Comment:
This was straight up not right before. It was getting the title of the tab in the current window, at the index of the _target_ tab.

Reply:
Hold up, is there a way to correct this then? Get the actual tab title of the target tab?

Reply:
Probs not trivially. Each TerminalWindow & TerminalPage is pretty agnostic to the existence of other windows at all. _maybe_ if the window that "caught" the moved pane raised the message, it could be done, but 🤷

Reply:
Ok, since we're still keeping the functionality, I think that won't be a problem for #15159. I'm just trying to avoid a Sev2 affecting our score again.